tests/fixtures.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+from kagglehub.config import set_kaggle_credentials
+
+class BaseTest(unittest.TestCase):
+    def setUp(self):
+        # Reset the global variable before each test
+        set_kaggle_credentials(None, None)
```

Comment:
Consider adding a `clear_kaggle_credentials` method which would set `_kaggle_credentials` to `None` instead.